lib/src/util/react_util.dart

```diff
@@ -72,7 +72,7 @@ class UiPropsMapView extends MapView
       throw UnimplementedError('@PropsMixin instances do not implement addUnconsumedDomProps');
 
   @override
-  void addTestId(String value, {String key = defaultTestIdKey}) =>
+  void addTestId(String? value, {String key = defaultTestIdKey}) =>
```

Comment:
Why would we want to allow this to be null?

Reply:
Pretty much same answer as https://github.com/Workiva/over_react/pull/856#discussion_r1394830339, for convenience when dealing with props like `String? barTestId`.